src/common/networking/connection.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
-#include <boost/asio.hpp>
-#include <boost/asio/ssl.hpp>
-#include <boost/asio/ts/buffer.hpp>
-#include <boost/asio/ts/internet.hpp>
+#include <asio.hpp>
+#include <asio/ssl.hpp>
+#include <asio/ts/buffer.hpp>
+#include <asio/ts/internet.hpp>
```

Comment:
I checked and there is a libasio-dev package for linux so we should be able to get away with just linking to that instead of the entire boost library.